examples/package/package.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
```

Comment:
Maybe call the example something better, it's an example of a package but it's also just a cassandra example.